src/main/io/osd.c

```diff
@@ -115,6 +115,8 @@ static uint32_t blinkBits[(OSD_ITEM_COUNT + 31)/32];
 static timeUs_t flyTime = 0;
 static uint8_t statRssi;
 
+bool adjustmentUpdateIdle = false;
```

Comment:
Why are you adding this flag? There is already a busy flag in `rc_adjustments.c`. That flag should be made available and used.

Reply:
replaced by a new function referencing to the available flag.